template/owt.m4

```diff
@@ -19,6 +19,7 @@ ARG SOURCE_DIR=ifelse(BUILD_DEV,enabled,/home/build/home,/home)
 ARG OWT_SDK_REPO=https://github.com/open-webrtc-toolkit/owt-client-javascript.git
 ARG OWT_BRANCH=4.3.x
 ARG DOWNLOAD_JSON_LINK="https://github.com/nlohmann/json/releases/download/v3.6.1/json.hpp"
+ENV LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/usr/local/ifelse(index(DOCKER_IMAGE,ubuntu),-1,lib64,lib/x86_64-linux-gnu):ifelse(index(DOCKER_IMAGE,xeon-),-1,/opt/intel/mediasdk/lib64,,)
```

Comment:
pls. fix the warning reported related to “lib64,,” -  Warning: excess arguments to builtin `ifelse' ignored